Neos.ContentRepository.Core/Classes/EventStore/EventPersister.php

```diff
@@ -24,22 +25,31 @@ public function __construct(
     }
 
     /**
+     * TODO Will be refactored via https://github.com/neos/neos-development-collection/pull/5321
      * @throws ConcurrencyException in case the expectedVersion does not match
      */
     public function publishEvents(ContentRepository $contentRepository, EventsToPublish $eventsToPublish): void
     {
         if ($eventsToPublish->events->isEmpty()) {
             return;
         }
+        $this->publishWithoutKetchup($eventsToPublish);
+        $contentRepository->catchUpProjections();
+    }
+
+    /**
+     * TODO Will be refactored via https://github.com/neos/neos-development-collection/pull/5321
+     * @throws ConcurrencyException in case the expectedVersion does not match
+     */
+    public function publishWithoutKetchup(EventsToPublish $eventsToPublish): CommitResult
```

Comment:
🍅

Reply:
refactored or not, that other PR is still Draft, and while I have full trust in bwaidelich, I don't like to bet on anything here, we should release, maybe it must be without subscription engine 🤷 who knows, but I don't really awnt ot releaes with ketchup. So please rename :)

Reply:
Thanks for pointing out how careful I did my review 🙈

Reply:
I guess I had 🍅 🍅 on my eyes.

Reply:
i renamed it again to the proper naming in https://github.com/neos/neos-development-collection/pull/5342/commits/dfd457360a13aee60ae9c22d9a19c5c4b5683dcb :D I had it corrected but was tempted as we had this fun sprint today and what would be a sprint day without Schabernack :D